### ISSUE TITLE
C1M4: Douglas' escape truck can be clipped through.

### ISSUE THREAD

**Issue by Shockwave508** [Realm667/WolfenDoom#812]
After taking out all of the Flakverling guns and escaping the dam, Douglas' escape vehicle can be clipped through from any and all angles, as if the model itself was a non-solid object.

**Comment by x3003** [comment]
Yes, you can go through the truck.

**Comment by Username-N00b-is-not-available** [comment]
BTW I've always wondered where Ryan and Douglas could find a General Motors truck after having parachuted in Germany... Replaced it with an Opel Blitz and fixed this issue.
Also there were some leftover scripts from lifts that previously existed on the map. The lifts were changed by @Ozymandias81 to portals, but the scripts were left in the ACS source. I checked the scripts and removed unused ones (1, 2, 5, 6).
